### DISCUSSION TITLE
I found cryptomining stuff in this program

### DISCUSSION THREAD

**Discussion by akatsuki600** [vzhd1701/gridplayer#120] [Q&A]
![crypto mining in my pc](https://user-images.githubusercontent.com/78849759/227116783-5775bc79-1a6d-4731-af82-a889e6865cf0.png)
This is what i found in the project folder, how is this allowed to be on github?

**Comment by potatofish** [comment]
> [screenshot] This is what i found in the project folder

Best I can tell from looking the src, Cryptodome & Crypto are the folders for the python cryptography libraries [pyCryptodome](https://github.com/Legrandin/pycryptodome) & [pyCrypt](https://www.pycrypto.org)

There are some references to those libraries in the src code, but I don't think I see any use of the libraries. I'm guessing means there's some use within one of the dependencies that I don't have installed yet. I'll review them once I get it going, hopefully the usage of those libraries will be above board. I think they probably will be.

Oh, and there is one cryptocurrency reference in the src, BUT it's only from the author's funding yml

**Comment by WilliUreta** [comment]
I mean, I'm no expert, but I don't think a crypto-scammer/stealer would name their folders as Crypto and Cryptodome. Given that there is an easily searchable Cryptodome project (https://pycryptodome.readthedocs.io/en/latest/src/introduction.html), the better question would be why is cryptographic stuff (sha256, MD5 and all the code inside these 2 folders) needed for this project.

**Comment by vzhd1701** [comment]
Cryptodome & Crypto directories belong to pycryptodome. Both yt-dlp and Streamlink that I use in GridPlayer depend on this library.
